### DISCUSSION TITLE
Wiz850io on non-eth boards

### DISCUSSION THREAD

**Discussion by phil-barrett** [phil-barrett/PicoCNC#18] [General]
It is possible to use the Wiz850io with non-eth PicoCNC. It requires connecting 3 wires in addition to the standard SPI wires. Unfortunately,  I am away from my PC for several weeks so can only provide text descriptions. I will fill in photos and diagrams when I return in March.

You use the SPI header pins except for the  CS pin. Connect 3.3V and Gnd to the same pins on the Wiz850io. Connect CK to SCLK. Connect TX to MOSI and RX to MISO.   

Now, you will need find 3 additional connections: CS, Int and Reset. 

CS: This comes from UART enable pin. Use the Raspi Pico GPIO 2 pin. You can solder a wire at the Pico socket on the bottom of the PicoCNC board. Do not use UART socket, I have not tested it with Voltage translation. Connect this to the SCNn pin on the Wiz850io. 

Int: Use the I2C strobe pin for this. It is marked as ST on the I2C header just above Pico socket. Connect it to the INTn pin on Wiz850io.  You may need to add a pull-up resistor to 3.3V. 4.7K ohms. This is on the 1.53eth board but, during development,  I did not use it and it seemed to work fine. 

Reset: Use the Aux7 pin on the PicoCNC and connect it to the RSTn pin on the Wiz850io. 

Be sure to build a version of grblHAL with ethernet selected. Select the Wiz550 option in the Networking pull-down  menu. Select Telnet, Websocket and FTP options as well.

You will need to find a way to mount the Wiz850io securely. 

I have written this mostly from memory so there could be errors. I will check this page daily so ask questions if you have trouble.

**Comment by JJTechman68** [comment]
Phil would a "W5500 SPI to LAN Ethernet Network Module TCP IP STM32 Interface 3.3V 5V for Arduino WIZ820io RC5" work on the version 1.52 board? If so are the connections the same as above, as well as the configuration in the webbuilder?

Thanks

**Reply by phil-barrett** [reply]
Unfortunately, not.  But I will give you a nice discount on the latest that does.  Contact me at my email address if interested.
